options.mk

```diff
@@ -15,12 +15,13 @@ ifeq ($(SIGN),ECC256)
     ./lib/wolfssl/wolfcrypt/src/ecc.o \
     ./lib/wolfssl/wolfcrypt/src/memory.o \
     ./lib/wolfssl/wolfcrypt/src/wc_port.o \
+    ./lib/wolfssl/wolfcrypt/src/wolfmath.o \
     ./lib/wolfssl/wolfcrypt/src/hash.o
-  CFLAGS+=-DWOLFBOOT_SIGN_ECC256 -DXMALLOC_USER
+  CFLAGS+=-D"WOLFBOOT_SIGN_ECC256"
   ifeq ($(WOLFTPM),0)
     CFLAGS+=-Wstack-usage=3888
   else
-    CFLAGS+=-Wstack-usage=6680
+    #CFLAGS+=-Wstack-usage=6680
```

Comment:
Maybe keep this on for DEBUG=1 build?

Reply:
Nope, that was mistake from initial work. The TI compiler doesn't recognize that flag, but I refactored and have it overridden for the R5.